### DISCUSSION TITLE
Welcome to WorldDynamics-Engine Discussions!

### DISCUSSION THREAD

**Discussion by stoleyourharvs** [thatguycy/WorldDynamics-Engine#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 1ByteBit** [comment]
Hello! I believe this plugin has a lot of potential and can't wait to see where it goes. Hopefully I don't get too annoying with my suggestions :)
